LibShifterBox/LibShifterBox.lua

```diff
@@ -410,17 +542,23 @@ local function _addEntryToList(list, key, value, replace, otherList, categoryId)
 end
 
 local function _moveEntriesToOtherList(sourceList, keys, destList, shifterBox)
+    local retVarLoop = false
+    local retVar = true
```

Comment:
What is this variable (retVar) used for? From what I can see you initialize it with 'true' and at some point maybe set it to 'false', but then nothing happens with it? :)

Or should the returned parameter be this one? (As found in eg. toRightButtonClicked)

Reply:
Whoops, the return variable should have been retVar instead of retVarLoop, like in toRightButtonClicked. Fixed it, thanks.